truenas_installer/install.py

```diff
@@ -42,14 +46,18 @@ async def install(disks, set_pmbr, authentication, post_install, sql, callback):
             raise InstallError(f"Command {' '.join(e.cmd)} failed:\n{e.stderr.rstrip()}")
 
 
-async def format_disk(device, set_pmbr, callback):
+async def wipe_disk(device, callback):
     if (result := await run(["wipefs", "-a", device], check=False)).returncode != 0:
         callback(0, f"Warning: unable to wipe partition table for {device}: {result.stderr.rstrip()}")
 
     # Erase both typical metadata area.
     await run(["sgdisk", "-Z", device], check=False)
```

Comment:
Furthermore, the double call to `sgdisk -Z` is completely unnecessary since `wipefs -a` does everything we need it to.